server/dba/migrations/47-addPermitCords.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
```

Comment:
This whole file needs indentation / formatting.

Reply:
@briandavidson - I formatted this one as well